src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,21 @@
+body {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+h2 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
```

Comment:
If the tags/classes have the same styles, they can be written all at once separated by a comma, so as not to duplicate the code.